brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Advanced Heading  - Added default width and color for separator.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/86165a51-3065-4302-9acb-fddb3834f1fe?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- For the old user we use % as the default unit. So keeping it the same added 12% looks good same as 80px 
Fixed
 - https://share.getcloudapp.com/X6uE7eRJ

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop the Creative Heading block.
- Enable Separator by setting style check width, color, and thickness.
- Frontend as well as on backend editor.
- Check 3rd issue mentioned by the Acceptance tester in round 2. It seems that it is fixed. Not able to replicate (Added screencast in card)

### Checklist:
